tests/integration/baseTestAPI.py

```diff
@@ -78,79 +69,36 @@ def setUpClass(cls):
             cgac = CGAC(cgac_code='000', agency_name='Example Agency')
 
             # set up users for status tests
-            def add_status_user(email, status_name, website_admin=False):
-                sess.add(UserFactory(
+            def add_user(email, name, username, website_admin=False):
+                user = UserFactory(
                     email=email, website_admin=website_admin,
                     affiliations=[UserAffiliation(
                         cgac=cgac,
                         permission_type_id=PERMISSION_TYPE_DICT['writer']
                     )]
-                ))
-            add_status_user('user@agency.gov', 'awaiting_confirmation')
-            add_status_user('realEmail@agency.gov', 'email_confirmed')
-            add_status_user('waiting@agency.gov', 'awaiting_approval')
-            add_status_user('impatient@agency.gov', 'awaiting_approval')
-            add_status_user('watchingPaintDry@agency.gov', 'awaiting_approval')
-            add_status_user(test_users['admin_email'], 'approved', True)
-            add_status_user(test_users['approved_email'], 'approved')
-            add_status_user('nefarious@agency.gov', 'denied')
+                )
+                user.salt, user.password_hash = getPasswordHash(user_password, Bcrypt())
+                user.name = name
+                user.username = username
```

Comment:
Small detail, but would it make sense to set these two attributes in the `UserFactory` constructor?